### PR TITLE
Get gas price from basescan

### PR DESCRIPTION
**Pull request by koutst**

### CODE REVIEW

File: src/reputation/views/withdrawal_view.py [hunk 1]
```diff
@@ -28,15 +28,19 @@
 from reputation.models import Withdrawal
 from reputation.permissions import AllowWithdrawalIfNotSuspecious
 from reputation.serializers import WithdrawalSerializer
-from researchhub.settings import ETHERSCAN_API_KEY, WEB3_PROVIDER_URL, WEB3_RSC_ADDRESS
+from researchhub.settings import (
+    BASESCAN_API_KEY,
+    ETHERSCAN_API_KEY,
+    WEB3_PROVIDER_URL,
+    WEB3_RSC_ADDRESS,
+)
```
Comment (gzurowski) [inline src/reputation/views/withdrawal_view.py:36]: Import `settings` not the settings themselves.

File: src/reputation/views/withdrawal_view.py [hunk 2]
```diff
@@ -214,11 +226,12 @@ def calculate_transaction_fee(self, network="ETHEREUM"):
             )
             json = res.json()
             gas_price = json.get("result", {}).get("SafeGasPrice", 40)
-            gas_limit = 120000.0
+
+        gas_limit = 120000.0
```
Comment (gzurowski) [inline src/reputation/views/withdrawal_view.py:230]: Nit: Does it make sense to add a comment or use a constant for this magic number?

**Review by gzurowski — APPROVED** [review]
LGTM 👍

File: src/reputation/views/withdrawal_view.py (hunk 1 above)
Comment (koutst) [reply src/reputation/views/withdrawal_view.py:36]: I have an upcoming PR that is based on this and fixes it.

**Pull request merged**